File: app/src/main/java/fr/scootop/data/model/tools/Match.java
package fr.scootop.data.model.tools;
import java.util.List;

import fr.scootop.data.model.configuration.Stade;
import fr.scootop.data.model.services.Event;
import fr.scootop.data.model.services.LocationWearable;
import fr.scootop.data.model.user.domain.Scoot;

public class Match {
	
    private Long id;
	
	private Event event;
	
	private Competition competition;
    
    private Team teamA;
    
    
    private Team teamB;
	
	private Stade stade;
	
	private Saison saison;

	private List<StatisticalSheet> statisticalSheetsTeamB;

    private List<StatisticalSheet> statisticalSheetsTeamA;
    

	private List<LocationWearable> wearables;
	
	private List <Scoot> scoots;
	
	
}

File: app/src/main/java/fr/scootop/data/model/tools/Match.kt
package fr.scootop.data.model.tools

import android.os.Parcelable
import fr.scootop.data.model.configuration.Stade
import fr.scootop.data.model.services.Event
import fr.scootop.data.model.services.LocationWearable
import fr.scootop.data.model.user.domain.Scoot
import kotlinx.android.parcel.Parcelize

@Parcelize
data class Match (
    var id: Long? = null,
    var event: Event? = null,
    var competition: Competition? = null,
    var teamA: Team? = null,
    var teamB: Team? = null,
    var stade: Stade? = null,
    var saison: Saison? = null,
    var statisticalSheetsTeamB: List<StatisticalSheet>? = null,
    var statisticalSheetsTeamA: List<StatisticalSheet>? = null,
    var wearables: List<LocationWearable>? = null,
    var scoots: List<Scoot>? = null
): Parcelable